src/internal/command.py

```diff
@@ -22,6 +23,8 @@ def build(cls, command_dict):
             print("Error parsing optional argument number {} for command \"{}\": {}".format(index + 1, command_dict['name'], str(e)))
             return None
 
+        arguments = tuple({RequiredArgument(x) for x in command_dict.get('arguments', [])})
```

Comment:
You should catch the KeyError here and output a descriptive error message like the one above for `options`